### PR TITLE
Feature/load current case

### PR DESCRIPTION
**Pull request by jwinding**
Loads the latest case into currentCase, and uses it to fill in values in the form (if they exist).

Also a small fix for the keyboard covering input fields. This needs more styling and possibly other logic alltogether, but we need something to make it usable in the meantime.

### CODE REVIEW

File: source/containers/Form/Form.js [hunk 1]
```diff
@@ -61,21 +70,29 @@ const {
...
                     };
                   }
 
                   case 'submit': {
                     return {
                       label: action.label,
-                      onClick: () => handleSubmit(onSubmit),
+                      onClick: () => {
+                        updateCaseInContext(formState.formAnswers, 'submitted');
+                        handleSubmit(onSubmit);
```
Comment (zilaei) [inline source/containers/Form/Form.js:85]: ESLint: 'handleSubmit' is not defined.

File: source/App.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ import { CaseProvider } from './store/CaseContext';
  */
 
 const App = () => {
-  if (Config.IS_STORYBOOK === 'true') {
+  if (Config.IS_STORYBOOK !== 'true') {
```
Comment (mysticlagoona) [inline source/App.js:23]: ===

File: source/components/molecules/ScreenWrapper.js [hunk 3]
```diff
@@ -14,7 +15,13 @@ const ScreenWrapper = props => {
 
   return (
     <ThemeProvider theme={theme}>
-      <Container style={style}>{children}</Container>
+      <Container
+        behavior={Platform.OS == 'ios' ? 'padding' : 'height'}
```
Comment (mysticlagoona) [inline source/components/molecules/ScreenWrapper.js:19]: Platform is not defined and ===. May be run Yarn run lint to see the errors

File: source/App.js (hunk 2 above)
Comment (jwinding) [reply source/App.js]: Fixed.

File: source/components/molecules/ScreenWrapper.js (hunk 3 above)
Comment (jwinding) [reply source/components/molecules/ScreenWrapper.js]: Fixed.

File: source/containers/Form/Form.js (hunk 1 above)
Comment (jwinding) [reply source/containers/Form/Form.js:85]: This looks different and is fixed in the newest version of this branch, so I'm saying that it's fixed.

**Pull request merged**